TYPO3.Flow/Classes/TYPO3/Flow/Core/Bootstrap.php

```diff
@@ -590,6 +602,12 @@ protected function ensureRequiredEnvironment()
                 exit(1);
             }
         }
+        if (!is_dir(FLOW_PATH_TEMPORARY) && !is_link(FLOW_PATH_TEMPORARY)) {
+            if (!@mkdir(FLOW_PATH_TEMPORARY, 0777, true)) {
```

Comment:
You're right - I'll add the umask magic from `createDirectoryRecursively()`. Using that method with a `try/catch` won't work because it does not use the shutup-operator leading to a fatal error